tests/integration/organizations/list.rs

```diff
@@ -6,6 +8,21 @@ fn command_organizations_list() {
         EndpointOptions::new("GET", "/api/0/organizations/?cursor=", 200)
             .with_response_file("organizations/get-organizations.json"),
     );
+
+    let region_response = format!(
+        "{{
+            \"regions\": [{{
+                \"name\": \"monolith\",
+                \"url\": \"{}\"
+            }}]
+        }}",
```

Comment:
This is slightly nicer with a raw string.
```suggestion
        r#"{{
            "regions": [{{
                "name": "monolith",
                "url": "{}"
            }}]
        }}"#,
```